package.json

```diff
@@ -21,17 +21,23 @@
     "typings"
   ],
   "scripts": {
-    "test": "nyc --extension --reporter=lcov --reporter=text-summary vows -- --spec test/*-test.js"
+    "build": "tsc",
```

Comment:
needs a `prepare` script that does the same thing as `build`

https://docs.npmjs.com/misc/scripts